mappostgis.cpp

```diff
@@ -2891,20 +2891,8 @@ msPostGISPassThroughFieldDefinitions( layerObj *layer,
       gml_type = "DateTime";
     }
 
-    char md_item_name[256];
-    snprintf( md_item_name, sizeof(md_item_name), "gml_%s_type", item );
-    if( msOWSLookupMetadata(&(layer->metadata), "G", "type") == NULL )
-      msInsertHashTable(&(layer->metadata), md_item_name, gml_type );
-
-    snprintf( md_item_name, sizeof(md_item_name), "gml_%s_width", item );
-    if( !gml_width.empty()
```

Comment:
Might need to check this outside of the updateGMLFieldMetadata function?

Reply:
According to https://stackoverflow.com/questions/8103342/calling-c-str-of-empty-string the c_str() will reurn an empty C string so no need to do a further check